Sources/Utilities/NotificationCenterDispatcher/KeyboardNotification.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  KeyboardNotification.swift
+//  PovioKit
+//
+//  Created by Borut Tomazin on 20/09/2021.
+//  Copyright © 2021 Povio Labs. All rights reserved.
+//
+
+import UIKit
+
+struct KeyboardNotification {
```

Comment:
If you want to expose this struct to the outside world it needs to be marked `public`; the same for all properties.

Reply:
Totally forget about this. Thank 👌